sgkit/stats/pedigree.py

```diff
@@ -203,6 +203,23 @@ def _raise_on_half_founder(
             raise ValueError("Pedigree contains half-founders")
 
 
+@numba_jit
+def _identify_founders(
+    parent: ArrayLike, tau: ArrayLike = None
+) -> ArrayLike:  # pragma: no cover
```

Comment:
BTW you should be able to omit the no cover pragma now that we are testing coverage of numba-jitted functions (#964).

Reply:
🥳... ☹️

Reply:
Thanks for fixing!